plugins/reporters/web-app-template/src/sagas/index.js

```diff
@@ -30,15 +30,14 @@ export function * loadOrtResultData() {
     const ortResultDataNode = document.querySelector('script[id="ort-report-data"]');
 
     if (ortResultDataNode) {
-        const { type: dataType } = ortResultDataNode;
+        const {textContent: ortResultDataNodeContents, type: ortResultDataNodeType} = ortResultDataNode;
```

Comment:
Is it style-wise really correct to no more have spaces after / before curly braces?

Reply:
No it isn't per ORT's eslint lint rules but wasn't caught as upgrade to ESLint causes some eslint plugins not to function properly and fail silently - downgraded to eslint 8.57 and now the above issue it get reported and fixed it by running `yarn lint --fix`.